### DISCUSSION TITLE
Future ready Talent

### DISCUSSION THREAD

**Discussion by Priya-1607** [github/india#267] [Students]
I have completed orientation session and industry session yesterday and it is in green but today it become red I use incongnito mode also but it is still in red should I fill the details again if anybody have any solution regarding this please reply me

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
